features/registerHealthCertificate.feature

```diff
@@ -0,0 +1,31 @@
+@RegistryService
+Feature: Register Health Certificate
+
+  @RegisterHealthCertificate
+  Scenario: Submit Valid Register Health Certificate Transaction
+    Given I make a POST request to /
+      And I set json body to the file at ./features/fixtures/registerHealthCertificate/Bundle-ProvideDDCCDocument.json
+    When I receive a response
+    Then I expect response should have a status 200
+      # Check that an entry exists in response for each entry in request, in same order
+      And I expect response should validate against the profile http://profiles.ihe.net/ITI/MHD/StructureDefinition/IHE.MHD.ProvideDocumentBundleResponse
+      And I expect a response entry exists for each request entry List, DocumentReference, Binary, Patient
```

Comment:
I would probably rewrite this step - you can use the pactum spec to retrieve the request body and write an expect similar to https://pactumjs.github.io/#/response-validation?id=adhoc that compares the request entries to response entries. The spec is in this.spec.

Reply:
It looks like this scenario is missing the profile check against http://profiles.ihe.net/ITI/MHD/StructureDefinition/IHE.MHD.ProvideDocumentBundleResponse.